Module 2. Basics. Node.js fundamental theory./task2/books.txt

```diff
@@ -0,0 +1,5 @@
+{"Book":"The Compound Effect","Author":"Darren Hardy","Amount":"5","Price":"9.48"}
+{"Book":"The 7 Habits of Highly Effective People","Author":"Stephen R. Covey","Amount":"4","Price":"23.48"}
+{"Book":"The Miracle Morning","Author":"Hal Elrod","Amount":"10","Price":"21.34"}
+{"Book":"Influence: The Psychology of Persuasion","Author":"Robert B. Cialdini","Amount":"4","Price":"12.99"}
+{"Book":"The ONE Thing","Author":"Gary Keller","Amount":"1","Price":"11.18"}
```

Comment:
I don't think we need the output file in repo

Reply:
I think it's good for result example. But ok I'll delete it if you want